source/includes/fact-older-version-limitations.rst

```diff
@@ -0,0 +1,43 @@
+- Writes that produce DDL events cannot occur on the source cluster during the 
+  migration. The following events cannot occur: 
+  
+  - ``collMod``
+  - ``create``
+  - ``createIndexes``
+  - ``drop``
+  - ``dropDatabase``
+  - ``dropIndexes``
+  - ``refineCollectionShardKey``  
+  - ``rename``
+  - ``reshardCollection``
+  - ``shardCollection``
+
+  This includes operations that may create new collections such as 
+  :dbcommand:`mapReduce`, :pipeline:`$out`, and :pipeline:`$merge`. This also 
+  includes collections created implicitly from inserts. Only writes that produce 
+  CRUD events can occur during the migration.
+
+  .. note:: 
+   
+     Writes that produce DDL events on source collections outside of the 
+     :ref:`namespace filter <c2c-filtered-sync>` are allowed.
+
+- ``geoHaystack`` indexes are not supported.
+
+- :ref:`/reverse <c2c-api-reverse>` endpoint is not supported. You can't 
+  enable the ``reversible`` option in the :ref:`/start <c2c-api-start>` request.
+
+- You can't enable the ``enableUserWriteBlocking`` option in the ``/start`` 
+  request. 
+
+- You can't enable the ``createSupportingIndexes`` :ref:`sharding parameter 
+  <c2c-api-start-sharding>`. 
```

Comment:
@ajayvijayakumar123 is there a workaround we can advise for users? It's tricky to digest what this parameter does if the reader isn't already familiar with it

Reply:
Yup. Let's say the user wants to shard on shard key `a:1`. Instead of enabling `createSupportingIndexes,` the user would need to create an index to support the shard key on the source.